### ISSUE TITLE
"HTTPSS" with ?json=true

### ISSUE THREAD

**Issue by dtregea** [Looskie/capybara-api#14]
## Overview

I've identified an error in the JSON response I receive. Specifically, URLs are incorrectly formatted with "httpss" instead of the correct "https" protocol when using json=true as a parameter.

## Steps to reproduce

1. Send a GET request to `https://api.capy.lol/v1/capybara?json=true`
2. Observe the "url" property in the returned JSON

## Expected behavior

The"url" property in the JSON response should start with "https", like so:

```json
{"success":true,"data":{"url":"https://api.capy.lol/v1/capybara/668","index":668,"width":1000,"height":666,"alt":"a capybara"}}
```

## Actual behavior
The "url" property in the JSON response incorrectly starts with "httpss":
```json
{"success":true,"data":{"url":"httpss://api.capy.lol/v1/capybara/668","index":668,"width":1000,"height":666,"alt":"a capybara"}}
```

**Comment by emilio-aburto** [comment]
I am observing the same problem.

**Comment by raxracks** [comment]
It seems to come from
https://github.com/Looskie/capybara-api/blob/a229627b2e91065610fd5526e9ac2de175f68adb/utils/helper.go#L24
Could solve this by either removing the "s" from the concatenation or checking if the protocol already ends with an "s".

**Comment by Looskie** [comment]
Hey guys! thanks for the report, should be fixed in [c0b54a8](https://github.com/Looskie/capybara-api/commit/c0b54a866193289f61ce8ff3a74cba09c28be4cd)!
